common/containers/Tabs/GenerateWallet/components/GenerateWalletPasswordComponent.jsx

```diff
@@ -45,15 +62,22 @@ class GenerateWalletPasswordComponent extends Component {
     this.setState(nextState);
   }
 
+  onClickGenerateFileGenerateWallet = () => {
+    this.genNewWallet(this.props.generateWalletPassword.values.password);
```

Comment:
This doesn't belong in the component, probably better in a saga. leaving in here for now while I discuss with @crptm about the best place for it.